app/controllers/advertisements_controller.rb

```diff
@@ -216,6 +216,8 @@ def referral_code
   end
 
   def ad_test?
+    return true if request.host == "localhost"
```

Comment:
We might also want to add an ngrok check.

Reply:
@hopsoft We could use `request.local?` but that caused a few tests to fail so I went with this safer approach. I think it was just due to the way the tests were set up though. Thoughts?

Reply:
I'd prefer `request.local?` if its not too difficult to patch up the tests.